src/components/content-box/_content-box.scss

```diff
@@ -17,6 +17,10 @@ $includeHtml: false !default;
         margin-top: - rhythm(0.5);
         margin-bottom: rhythm(0.5);
       }
+
+      &--large ~ .mint-content-box__actions--large {
```

Comment:
Both yes and no.
Yes - as we need to squash some space for good looks. This space is caused by the fact that header-secondary falls out of rhythm if it is wrapped (which  is a real case for us).

No - because it's application should be limited to as few places as possible. 
This is the abstraction leak and creates implicit context. But we have to make this tradeoff.